server/src/main/java/com/exacaster/lighter/rest/SessionParameters.java

```diff
@@ -0,0 +1,25 @@
+package com.exacaster.lighter.rest;
+
+import com.exacaster.lighter.application.SubmitParams;
+import io.micronaut.core.annotation.Introspected;
+
+import java.util.List;
+import java.util.Map;
+
+@Introspected
+public class SessionParameters extends SubmitParams {
```

Comment:
maybe lets name it `SessionSubmitParams`, to align with `SubmitParams` name?

Reply:
can do. nevertheless `SubmitParams` suggests parameters to be submitted to Spark session but in this case `SessionParameters ` is about submit parameters and lighter specific configuration of session. WDYT ?